stacks/api/passport/identity/src/utility.ts

```diff
@@ -8,3 +10,83 @@ export function attachIdentityAccess(request: any) {
   }
   return request;
 }
+
+export function provideSettingsFinalizer(identityService: IdentityService) {
+  return (previousSchema: any, currentSchema: any) => {
+    let updatedFields = findUpdatedFields(
+      previousSchema.identity.attributes.properties,
+      currentSchema.identity.attributes.properties,
+      [],
+      ""
+    );
+
+    if (updatedFields.length < 1) return;
+
+    let unsetFields = updatedFields.reduce((acc, current) => {
+      let fieldName = "attributes." + current;
+      acc = {...acc, [fieldName]: ""};
+      return acc;
+    }, {});
+
+    return identityService.updateMany({}, {$unset: unsetFields});
+  };
+}
+
+export function findUpdatedFields(
```

Comment:
@thesayyn I know these codes are copied from bucket utilities and it should not be duplicated. Where should we put these common codes which are common for only bucket and identity?

Reply:
Can we hold this PR this we land bucket rules? only then I could think of something to about this.